Lib/test/test_ast/test_ast.py

```diff
@@ -417,6 +417,18 @@ def test_field_attr_existence(self):
                 if isinstance(x, ast.AST):
                     self.assertIs(type(x._fields), tuple)
 
+    def test_dynamic_attr(self):
+        for name, item in ast.__dict__.items():
+            # constructor has a different signature
+            if name == 'Index':
+                continue
+            if self._is_ast_node(name, item):
+                x = self._construct_ast_class(item)
+                # Random attribute assignment is allowed
```

Comment:
```suggestion
                # Custom attribute assignment is allowed
```

It's not "random" (I know you copied the comment which may have been mine to begin with...)